python-package/xgboost/dask.py

```diff
@@ -735,8 +735,9 @@ def train(client, params, dtrain, *args, evals=(), early_stopping_rounds=None,
     '''
     _assert_dask_support()
     client = _xgb_get_client(client)
+    global_config = config.get_config()
```

Comment:
Please collect all changes about global configuration and submit a separate pull request.

Reply:
It's WIP.  The fixed is already in master branch and I will remove it here.  I mentioned that I found the bug during debugging,  I was debugging this PR. ;-)